ftw/activity/browser/templates/activity_raw.pt

```diff
@@ -2,6 +2,7 @@
 
   <div class="activity">
     <div class="events"
+         tal:attributes="data-total python:len(view._lookup())"
```

Comment:
This might resolve lazyness: it probably loads all data from the Database, which would be bad.
I need to investigate that first.